docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -148,11 +150,15 @@ Your synthetics scripts may require the use of passwords or other sensitive secr
 
 [WARNING]
 ====
-Because synthetics scripts have no limitations, a malicious script author could write a
-synthetics journey that exfiltrates `params` and other data at runtime.
+Please note that params are viewable in plain-text by administrators and other users with "all" privileges for
+the synthetics app.
+Additionally, note that synthetics scripts have no limitations on accessing these values, and  a malicious script author could write a
+synthetics journey that exfiltrates `params` and other data at runtime. 
 Do *not* to use truly sensitive passwords (for example, an admin password or a real credit card)
 in *any* synthetics tools.
 Instead, set up limited demo accounts, or fake credit cards with limited functionality.
+If you want to limit access to parameters ensure that that users who are not supposed to access those values do not have "all" privileges
```

Comment:
```suggestion
If you want to limit access to parameters ensure that users who are not supposed to access those values have "read" privileges
```

Reply:
I'm worried that this isn't quite right in that users with `read` privileges can still view a lot of network request metadata including request headers that might contain credentials. Do you agree we need to document / caveat this? I generally think that we should advise folks to think of it as more binary, if you can view the monitor you may be able to view credentials. Even if that isn't true all of the time, it is some of the time.

Your thoughts?

Reply:
Makes sense. Lets keep it the way you proposed.